backends/dpdk/dpdkArch.cpp

```diff
@@ -1624,16 +1624,16 @@ const IR::Node* CopyMatchKeysToSingleStruct::preorder(IR::Key* keys) {
     } else {
         structure->table_type_map.emplace(table->name.name, InternalTableType::WILDCARD);
     }
+
```

Comment:
new line?

Reply:
I think this is OK for readablilty. No issues reported by clang-format or cpplint as well.